packages/lektor_atom_plugin/lektor_atom.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+
+"""
+This is an updated copy of lektor-atom plugin
+wich replace deprecated Werkzeug's atom with Pelican's feedgenerator .
+"""
```

Comment:
Minor typos: "which replaces the deprecated Werkzeug's atom with Pelican's feedgenerator."